entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -115,6 +115,17 @@ class EntityService(
         }
     }
 
+    fun publishDeletionEvent(entityId: String) {
+        getSerializedEntityById(entityId)?.also {
+            val entityEvent = EntityEvent(
+                operationType = EventType.DELETE,
+                entityId = entityId,
+                payload = it
```

Comment:
is it necessary to send the payload ?

Reply:
I don't know

Reply:
I'll remove it as it is useless for deletion